### ISSUE TITLE
Scrolling/Swiping the submenu doesn't work in mobile when "Use sliding submenu" = false

### ISSUE THREAD

**Issue by ArturJo** [FrDH/mmenu-js#1248]
If you **switch off** the Option "Use sliding submenu" , you **cannot swipe** up or down a longer submenu in mobile view. Even on your demo website in playground. 
See the screenshot https://ibb.co/qy8WjMq 
Please open up all submenu and then try to drag the design-link and try to swipe up. I mean, on a mobile device like iPhone. It does not work. 
How can I fix it?

**Comment by MikeyJN** [comment]
Yes I'm also experiencing this on Android (Samsung Galaxy phone) on both Google Chrome and also the built-in native browser, so possibly a bug rather than a browser issue?

**Comment by MikeyJN** [comment]
Hi,

Any update with regard to this issue as to when a fix may become available please? I have a new project I'm working on and would love to be able to use the menu without the sliding submenus!!:)

Many thanks,

Mike

**Comment by MikeyJN** [comment]
Just wondered if anyone has managed to workout a fix for this yet?

FrDH in a couple of other recent issues I see you've mentioned about an upcoming major update. Do you have a time frame for this and if so, do you think the issue above be resolved in it?
 
many thanks!!:)

**Comment by FrDH** [comment]
I actually just finished updating the docs, examples and tutorials, so probably somewhere this week :)

**Comment by MikeyJN** [comment]
Thank-you so much for updating - now working perfectly!!:)

Just in case you're unaware, I've been having a look at the updated website and noticed that the animated burger icons aren't animating when clicked-on: https://mmenujs.com/mburger/ and also the documentation for mhead.js throws-up a 404: https://mmenujs.com/mhead

Thanks again so much!!:)

**Comment by FrDH** [comment]
Thanks for the heads up, I fixed the examples for mburger.
I removed the docs for mhead as I intend to abandon it. I'll see if there are references to the page that need to be removed.

**Comment by MikeyJN** [comment]
Sorry just one little 'non-issue' issue I've noticed is that when on a **mobile device** and the drop-downs are expanded and exceed the page/device height, you obviously get a scrollbar for the overflow. However, when you scroll and then stop scrolling, the scrollbar disappears only to then imedediatly reappear again causing what looks like for it to flicker (obviously not really a 'flicker', just the scroll-bar turning off and then straight back on again). This only happens on mobile devices and not desk-top (unless viewing in mobile view within the browsers dev tools). 

Any idea if there is a fix to either make it stay on or go off until scrolled again? I've tried various scrollbar related CSS properties, but they do nothing!!

Many thanks,

Mike
